### PR TITLE
add-stacked-layout

### PR DESCRIPTION
**Pull request by nullman**
Added master-stack layout.

### CODE REVIEW

**Comment by nullman** [comment]
See changes to the man page for details and options. Basically, you can now toggle between tiled, stacked, and manacle layouts.

File: doc/bspwm.1 [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 .\" Generator: DocBook XSL Stylesheets vsnapshot <http://docbook.sf.net/>
-.\"      Date: 04/05/2023
+.\"      Date: 01/01/1980
 .\"    Manual: Bspwm Manual
-.\"    Source: Bspwm 0.9.10-49-gc58c67a
+.\"    Source: Bspwm 0.9.10
 .\"  Language: English
 .\"
-.TH "BSPWM" "1" "04/05/2023" "Bspwm 0\&.9\&.10\-49\-gc58c67a" "Bspwm Manual"
+.TH "BSPWM" "1" "01/01/1980" "Bspwm 0\&.9\&.10" "Bspwm Manual"
```
Comment (Tectu) [inline doc/bspwm.1:10]: That looks fishy :)

File: doc/bspwm.1 [hunk 2]
```diff
@@ -2,12 +2,12 @@
 .\"     Title: bspwm
 .\"    Author: [see the "Author" section]
 .\" Generator: DocBook XSL Stylesheets vsnapshot <http://docbook.sf.net/>
-.\"      Date: 04/05/2023
+.\"      Date: 01/01/1980
```
Comment (Tectu) [inline doc/bspwm.1:5]: That looks fishy :)

File: doc/bspwm.1 (hunk 1 above)
Comment (nullman) [reply doc/bspwm.1:10]: The act of running "make doc" generated that change. I could always revert that line.

File: doc/bspwm.1 (hunk 2 above)
Comment (nullman) [reply doc/bspwm.1:5]: Same as above. The act of running "make doc" generated that change. I could always revert that line.

**Comment by nullman** [comment]
Resizing doesn't currently work, so I'm still working on that. (I forgot that I never got around to it.)

**Comment by nullman** [comment]
Closing this PR as I have found a bug in addition to needing to support resizing. Also, I want to do a refactor. Thanks to those that took a look.

**Comment by og900aero** [comment]
Is there no solution to this? But it is something that is sorely lacking.

**Comment by nullman** [comment]
> Is there no solution to this? But it is something that is sorely lacking.

I would like to get back to working on my PR as I also would like a master-stack mode for BSPWM. (I've simply had other priorities on my time.)

**Pull request closed without merging**